src/WebJobs.Script/BindingExtensionBundle/ExtensionBundleOptionsSetup.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Azure.WebJobs.Script.Configuration;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using NuGet.Packaging;
+using NuGet.Versioning;
+
+namespace Microsoft.Azure.WebJobs.Script.BindingExtensionBundle
+{
+    internal class ExtensionBundleOptionsSetup : IConfigureOptions<ExtensionBundleOptions>
+    {
+        private readonly IConfiguration _configuration;
+
+        public ExtensionBundleOptionsSetup(IConfiguration configuration)
+        {
+            _configuration = configuration;
+        }
+
+        public void Configure(ExtensionBundleOptions options)
+        {
+            IConfigurationSection jobHostSection = _configuration.GetSection(ConfigurationSectionNames.JobHost);
+            var extensionBundleSection = jobHostSection.GetSection(ConfigurationSectionNames.ExtensionBundle);
+
+            // value of extension bundle property is set to null
+            if (extensionBundleSection.Value == string.Empty)
+            {
```

Comment:
Is this needed? I see you have validation logic below

Reply:
I have added this when extensionBundle is set to null. `'extensionBundle': null'`. It shows up as empty string in the options. The check below is to ignore validation extensionBundle is not present at all.